src/explore-education-statistics-admin/src/contexts/ConfigContext.tsx

```diff
@@ -33,9 +33,21 @@ export function useConfig(): Config {
 // Config provider for testing
 
 const defaultTestConfig: Config = {
-  AppInsightsKey: '',
-  PublicAppUrl: 'http://localhost',
-  PermittedEmbedUrlDomains: ['https://department-for-education.shinyapps.io'],
+  appInsightsKey: '',
+  publicAppUrl: 'http://localhost',
+  permittedEmbedUrlDomains: ['https://department-for-education.shinyapps.io'],
```

Comment:
Hey, yeah we're just using `https://dfe-analytical-services.github.io` so that we can embed stuff in our Robot tests, so I wouldn't add it here I reckon.